enterprise/frontend/src/metabase-enterprise/transforms-python/components/PythonTransformEditor/PythonEditorResults/PythonEditorResults.tsx

```diff
@@ -55,7 +99,7 @@ function EmptyState() {
   const keyboardShortcut = getRunQueryShortcut();
 
   return (
-    <Flex h="100%" align="center" justify="center" className={S.empty}>
+    <Flex h="100%" align="center" justify="center">
```

Comment:
tip: mantine has a cute little `<Center />` component that is a shorthand for this